dashboard/settings.py

```diff
@@ -410,7 +410,11 @@
 
 # Method to show the user, if they're authenticated and superuser
 def show_debug_toolbar(request):
-    return DEBUG and request.user and request.user.is_authenticated and request.user.is_superuser
+    try:
+        return DEBUG and request.user and request.user.is_authenticated and request.user.is_superuser
+    except:
+        # If there's an exception here just don't show it
```

Comment:
Should the exception be logged at least?

Reply:
Yeah we could, I don't 100% remember what problem I saw here was but there was an exception being thrown when the user wasn't logged in from somewhere in Django and I just couldn't get it to go away. I'll except it but still log it for now. It will only come up when DEBUG is enabled, which should only be on our test servers.

Reply:
I'll just remove this, I don't think it's necessary to have in here and I don't remember why I added it.